eclipse-collections/src/main/java/org/eclipse/collections/impl/list/fixed/AbstractMemoryEfficientMutableList.java

```diff
@@ -124,6 +127,33 @@ public void clear()
         throw new UnsupportedOperationException("Cannot clear a fixed size list: " + this.getClass());
     }
 
+    /**
+     * This method checks if comparator is null and use a ComparableComparator if it is.
+     *
+     * @since 10.0
+     */
+    @Override
+    public void sort(Comparator<? super T> comparator)
+    {
+        this.insertionSort(Comparators.comparableComparatorIfNull(comparator));
```

Comment:
Do you have a JMH test for this? The extra virtual method calls from using a comparator may hurt performance.

Reply:
No. This code is only for FixedSizedLists of size 2 to 6.  I can try and measure it if you think it is important.